src/main/java/at/hannibal2/skyhanni/features/minion/MinionFeatures.kt

```diff
@@ -199,10 +199,12 @@ class MinionFeatures {
     }
 
     private fun removeBuggedMinions() {
+        if (!IslandType.PRIVATE_ISLAND.isInIsland()) return
         val minions = minions ?: return
 
         val removedEntities = mutableListOf<LorenzVec>()
         for (location in minions.keys) {
+            if (location.distanceToPlayer() > 30) continue
```

Comment:
i feel it should be a bit further than this, as this is only 5.5 blocks

Reply:
this checks for exactly 30 blocks, you confuse it with distanceSqToPlayer